adcc/backends/test_backends_pcm.py

```diff
@@ -0,0 +1,206 @@
+import pytest
+import unittest
+import itertools
+import adcc
+import adcc.backends
+import os
+
+from adcc.misc import expand_test_templates
+from adcc.testdata import static_data
+from adcc.testdata.cache import psi4_data, pyscf_data
+from numpy.testing import assert_allclose
+from adcc.exceptions import InputError
+
+from adcc.adc_pp.environment import block_ph_ph_0_pcm
+from adcc.AdcMatrix import AdcExtraTerm
+
+backends = [b for b in adcc.backends.available()
+            if b in ["psi4", "pyscf"]]
+basissets = ["sto-3g", "cc-pvdz"]
+methods = ["adc1", "adc2", "adc3"]
+
+
+@pytest.mark.skipif(len(backends) == 0, reason="Psi4 and Pyscf backends not found.")
```

Comment:
```suggestion
@pytest.mark.skipif(len(backends) == 0, reason="No backend for PCM available.")
```

Reply:
and I could also fix the method to adc1, because atm I only test for adc1 (although I could also adc2 etc. tests for ptLR)

Reply:
I'd say fix it to ADC1 outside, because it tests all the necessary machinery + we don't have any reference data for LR 👍